core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -632,6 +632,15 @@ cairo.sql.copy.root=import
 # When disabled, SQL executed by materialized view refresh job always runs single-threaded.
 #cairo.mat.view.parallel.sql.enabled=true
 
+# Defines the desired number of base table rows to be scanned by queries run by the materialized view refresh job.
+#cairo.mat.view.rows.per.query.estimate=10000000
+
+# Minimal time interval used by the refresh job when running materialized view queries.
+#cairo.mat.view.min.refresh.interval=1m
```

Comment:
I'd choose 1s instead of 1m. 
As far as I understand, `1m` means that when the view sample by is `1s`, the refresh query will run for `1m` or a bigger window. Is this correct?

1s mat views seem to be a reasonably popular choice to me, subsecond more like exotics

Reply:
> As far as I understand, `1m` means that when the view sample by is `1s`, the refresh query will run for `1m` or a bigger window. Is this correct?

Yes, that's correct.

I chose `1m` because it should be an order of magnitude faster to walk a year with minute intervals. The downside is not that big - in the worst case we're speaking of additional 59 seconds before the initial [min_ts, max_ts] interval and 59 seconds after it to be scanned. WDYT?